### DISCUSSION TITLE
Sonoff Basic: after a power cut how to always switch off the relay

### DISCUSSION THREAD

**Discussion by PPHIL972** [arendst/Tasmota#12210] [Q&A] [answered]
Hello, i have a problem: I hacked a Sonoff Basic and when there is a power cut, the state of the relay before the cut remains the same. How to make its state always off after the return of the current? 
Thank you very much for your help.

**Comment by sfromis** [comment, marked as answer]
[PowerOnState](https://tasmota.github.io/docs/Commands/#poweronstate)
[SetOption0](https://tasmota.github.io/docs/Commands/#setoption0) allows you to not save power state.
